### ISSUE TITLE
Bug in exchange.fetch_open_orders(market) where market is either BSV or BCH

### ISSUE THREAD

**Issue by asafravid** [ccxt/ccxt#5271]
- OS: `Windows`
- Programming Language version: `Python 3.7`
- CCXT version: `ccxt-1.18.661`
- Exchange: `bit2c`
- Method: `fetch_open_orders()`

Consider the following code:

    def scan_my_open_orders(exchange, markets):
        my_open_orders = {}
        for market in markets:
            open_orders = exchange.fetch_open_orders(market)
            print('[scan_my_open_orders] Scanning {} open {} orders'.format(len(open_orders), market))
            if len(open_orders):
                ...

Resulting output:

    [scan_my_open_orders] Scanning 2 open BTC/NIS orders
    [scan_my_open_orders] Scanning 1 open ETH/NIS orders
    [scan_my_open_orders] Scanning 0 open BCH/NIS orders  <--- BUG
    [scan_my_open_orders] Scanning 1 open LTC/NIS orders
    [scan_my_open_orders] Scanning 2 open ETC/NIS orders
    [scan_my_open_orders] Scanning 3 open BTG/NIS orders
    [scan_my_open_orders] Scanning 0 open BSV/NIS orders  <--- BUG
    [scan_my_open_orders] Scanning 9 open GRIN/NIS orders

Analysis:

1. I verified I have open orders for `BCH` (`bit2c` refers to it as `BCHABC`) and `BSV` (`bit2c` refers to it as `BCHSV`). 
2. `markets` parameter is the markets returned by `exchange.load_markets()`

Hence, fetching the open orders fails to return the open orders of those 2 coins.

What I suspect is that there is some mix-up between the following pairs:
1. `BCHABC <-> BCH`
2. `BCHSV <-> BSV`

While you check your side, I will debug the contents of `fetch_open_orders()` and add some print outs to see what exactly goes wrong there

Thanks

**Comment by asafravid** [comment]
Update:

- The `market['id']` seems to be correctly produced within `fetch_open_orders()`.
- Since I don't have the code for `privateGetOrderMyOrders()` (or am I missing something here?), I cannot check what went wrong in `fetch_open_orders()`

Debug Code:

    def fetch_open_orders(self, symbol=None, since=None, limit=None, params={}):
        if symbol is None:
            raise ArgumentsRequired(self.id + ' fetchOpenOrders() requires a symbol argument')
        self.load_markets()
        market = self.market(symbol)
        request = {
            'pair': market['id'],
        }
        print('***** [fetch_open_orders] request = {}'.format(request))
        response = self.privateGetOrderMyOrders(self.extend(request, params))
        orders = self.safe_value(response, market['id'], {})
        asks = self.safe_value(orders, 'ask', [])
        bids = self.safe_value(orders, 'bid', [])
        return self.parse_orders(self.array_concat(asks, bids), market, since, limit)

Output Code:

     ***** [fetch_open_orders] request = {'pair': 'BtcNis'}
     [scan_my_open_orders] Scanning 2 open BTC/NIS orders
     ***** [fetch_open_orders] request = {'pair': 'EthNis'}
     [scan_my_open_orders] Scanning 1 open ETH/NIS orders
     ***** [fetch_open_orders] request = {'pair': 'BchAbcNis'}
     [scan_my_open_orders] Scanning 0 open BCH/NIS orders
     ***** [fetch_open_orders] request = {'pair': 'LtcNis'}
     [scan_my_open_orders] Scanning 1 open LTC/NIS orders
     ***** [fetch_open_orders] request = {'pair': 'EtcNis'}
     [scan_my_open_orders] Scanning 2 open ETC/NIS orders
     ***** [fetch_open_orders] request = {'pair': 'BtgNis'}
     [scan_my_open_orders] Scanning 3 open BTG/NIS orders
     ***** [fetch_open_orders] request = {'pair': 'BchSvNis'}
     [scan_my_open_orders] Scanning 0 open BSV/NIS orders
     ***** [fetch_open_orders] request = {'pair': 'GrinNis'}
     [scan_my_open_orders] Scanning 9 open GRIN/NIS orders

**Comment by kroitor** [comment]
> Since I don't have the code for privateGetOrderMyOrders() (or am I missing something here?)

Yes, you're missing this entire section of the Manual: https://github.com/ccxt/ccxt/wiki/Manual#implicit-api-methods )

Can you please share the `verbose` **request** and **response**? Otherwise it is not clear what exactly is wrong. Please, follow these guides: 

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue
- https://github.com/ccxt/ccxt/wiki/Manual#troubleshooting

```Python
# ...
exchange.load_markets()
exchange.verbose = true  # ←--------  ADD THIS AFTER load_markets BEFORE fetch_open_orders
def scan_my_open_orders(exchange):
    for symbol in exchange.symbols:
        open_orders = exchange.fetch_open_orders(symbol)
        print(str(len(open_orders) + ' ' + symbol + ' open orders')
```

**Comment by kroitor** [comment]
More on implicit API methods here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#implicit-api-methods

**Comment by asafravid** [comment]
Ok, I deleted the `Key` and `Sign` from the outputs:

Code (added `verbose` after `load_markets()`):

    def scan_my_open_orders(exchange, markets):
        my_open_orders = {}
        for market in markets:
            open_orders = exchange.fetch_open_orders(market)
            print('[scan_my_open_orders] ' + str(len(open_orders)) + ' ' + market + ' open orders')
            print('[scan_my_open_orders] Scanning {} open {} orders'.format(len(open_orders), market))
            if len(open_orders):
                my_open_orders_for_this_market = []
                for order in open_orders:
                    my_order = {
                            'symbol': order['symbol'],
                            'side': order['side'],
                            'amount': order['amount'],
                            'price': order['price']
                    }
                    my_open_orders_for_this_market.append(my_order)
                my_open_orders[market] = my_open_orders_for_this_market

        return my_open_orders

Outputs:

    ***** [fetch_open_orders] request = {'pair': 'BtcNis'}
    
    Request: GET https://bit2c.co.il/Order/MyOrders {'Content-Type': 'application/x-www-form-urlencoded', 'key': 'REMOVED', 'sign': 'NGNa1f8gpdmDVnAZmiImQbGc7zXaWOkKYoE07+yOboV2VWCM/T+==', 'User-Agent': 'python-requests/2.22.0', 'Accept-Encoding': 'gzip, deflate'} nonce=1560112141&pair=BtcNis
    Response: GET https://bit2c.co.il/Order/MyOrders 200 {'Date': 'Sun, 09 Jun 2019 20:29:01 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '182', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=; expires=Mon, 08-Jun-20 20:29:01 GMT; path=/; domain=.bit2c.co.il; HttpOnly; Secure, .ASPXAUTH=4A409D83D1E646D96E188072BB27DF5771A4DD7BACDDDE887FE53BD35ED1CEB3E08C41277E456A1FDE5A19C95C94656D94CDC674AEDA53573F7169466A4FE69AE30ABB4F2EE89177CE5886CB90EDEBE692213ED446A0BE7D9BEA80423C3B48C5184BFA608C4835EBB51E50D2CC320954C6FCD775BCC43DF2C831EC9B2D940074FD857351739A041D5E1F2D2B3D239C8E; path=/; HttpOnly', 'Cache-Control': 'private, s-maxage=0', 'Content-Encoding': 'gzip', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4e45daf4180997de-FRA'} {"BtcNis":{"ask":[{"created":1559503278,"type":1,"order_type":0,"amount":0.00100000,"price":30599.90000000,"stop":0.00000000,"id":169969387,"initialAmount":0.00100000}],"bid":[{"created":1559623607,"type":0,"order_type":0,"amount":0.01000000,"price":25000.00000000,"stop":0.00000000,"id":170624959,"initialAmount":0.01000000}]}}
    [scan_my_open_orders] 2 BTC/NIS open orders
    [scan_my_open_orders] Scanning 2 open BTC/NIS orders
    ***** [fetch_open_orders] request = {'pair': 'EthNis'}
    
    Request: GET https://bit2c.co.il/Order/MyOrders {'Content-Type': 'application/x-www-form-urlencoded', 'key': 'REMOVED', 'sign': 'Oo3aulEc8OatEABiuofMKVrSTO3IHcOuMrJOo/+pR5Q52dkc+/K/MixAgoe6kEZWQ==', 'User-Agent': 'python-requests/2.22.0', 'Accept-Encoding': 'gzip, deflate'} nonce=1560112144&pair=EthNis
    Response: GET https://bit2c.co.il/Order/MyOrders 200 {'Date': 'Sun, 09 Jun 2019 20:29:04 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '147', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=; expires=Mon, 08-Jun-20 20:29:04 GMT; path=/; domain=.bit2c.co.il; HttpOnly; Secure, .ASPXAUTH=464CF86AD651971A023CDD58B2D37F7A6D32A04C3A0D88CB5824CA10F5192F9A1D168EAD4FE8B0F7C00DD5612CB724348BFE72CA84C14993C535FC5A8224108817D95806EDF61C26DDAF4385B0AF1632CC6CD94C94CA9F0F7D72BB08D0F70C0D2AA8D0EEF0B2108803D011F6B811EAD59B71B0C8DF15C940FFA5C30871B7691922727585DDE6AA817DBC420E05F8D23F; path=/; HttpOnly', 'Cache-Control': 'private, s-maxage=0', 'Content-Encoding': 'gzip', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4e45db06e8ff97de-FRA'} {"EthNis":{"ask":null,"bid":[{"created":1559623629,"type":0,"order_type":0,"amount":0.25000000,"price":800.00000000,"stop":0.00000000,"id":170625147,"initialAmount":0.25000000}]}}
    [scan_my_open_orders] 1 ETH/NIS open orders
    [scan_my_open_orders] Scanning 1 open ETH/NIS orders
    ***** [fetch_open_orders] request = {'pair': 'BchAbcNis'}
    
    Request: GET https://bit2c.co.il/Order/MyOrders {'Content-Type': 'application/x-www-form-urlencoded', 'key': 'REMOVED', 'sign': 'CIp6e3vNgHuZeoC61HK/asdnm0CR7DCvZPE/N/==', 'User-Agent': 'python-requests/2.22.0', 'Accept-Encoding': 'gzip, deflate'} nonce=1560112147&pair=BchAbcNis
    Response: GET https://bit2c.co.il/Order/MyOrders 200 {'Date': 'Sun, 09 Jun 2019 20:29:07 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '224', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=; expires=Mon, 08-Jun-20 20:29:07 GMT; path=/; domain=.bit2c.co.il; HttpOnly; Secure, .ASPXAUTH=7CF639A42C4A45E8E2109EE6011E898E5CC9D63174034E8FB250F226BE3363283473589825B0A4BA3F078E481AB51EC5710D30A0E2B5668F11A263F919247886A0E2A3561E765F696785128E22CC44ED3625CA2E16EFB357BA22EA6EC2EA8D13EE01056178F2DCCE641A15B0192FE199220A929BDBD145D39535E0390FE203C7FF3C1A543AA31BA3A24FA7D1EF4EACA3; path=/; HttpOnly', 'Cache-Control': 'private, s-maxage=0', 'Content-Encoding': 'gzip', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4e45db19ab4197de-FRA'} {"BchabcNis":{"ask":[{"created":1559881543,"type":1,"order_type":0,"amount":0.05000000,"price":1402.01000000,"stop":0.00000000,"id":172288301,"initialAmount":0.05000000},{"created":1559672223,"type":1,"order_type":0,"amount":0.07500000,"price":1617.16000000,"stop":0.00000000,"id":170951278,"initialAmount":0.07500000}],"bid":[{"created":1559881497,"type":0,"order_type":0,"amount":0.05000000,"price":1072.63000000,"stop":0.00000000,"id":172288028,"initialAmount":0.05000000},{"created":1559929546,"type":0,"order_type":0,"amount":0.05000000,"price":1112.49000000,"stop":0.00000000,"id":172557238,"initialAmount":0.05000000}]}}
    [scan_my_open_orders] 0 BCH/NIS open orders
    [scan_my_open_orders] Scanning 0 open BCH/NIS orders
    ***** [fetch_open_orders] request = {'pair': 'LtcNis'}
    
    Request: GET https://bit2c.co.il/Order/MyOrders {'Content-Type': 'application/x-www-form-urlencoded', 'key': 'REMOVED', 'sign': '==', 'User-Agent': 'python-requests/2.22.0', 'Accept-Encoding': 'gzip, deflate'} nonce=1560112150&pair=LtcNis
    Response: GET https://bit2c.co.il/Order/MyOrders 200 {'Date': 'Sun, 09 Jun 2019 20:29:10 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '144', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=; expires=Mon, 08-Jun-20 20:29:10 GMT; path=/; domain=.bit2c.co.il; HttpOnly; Secure, .ASPXAUTH=ABCB33533D8860419AA007CA34A6E170AE2D4D0DC0315A32F8D7890EFA43609411BA04D454BA305CCE920E85DAED47385A903BF36906A56E1E28BFFF574CFC02D72E9A4B70DBADE5365057EB9BA7E07E9457BB5B6A37EAA1DC795B65AF5C058038212A4F7FAD43F4162CEFB4F2D4B66C9FDBE4F86212C77478C4E4D03589CA244D253810ACF8B6EB64FC2CC0423C50DE; path=/; HttpOnly', 'Cache-Control': 'private, s-maxage=0', 'Content-Encoding': 'gzip', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4e45db2c7d5197de-FRA'} {"LtcNis":{"ask":null,"bid":[{"created":1559623669,"type":0,"order_type":0,"amount":0.50000000,"price":345.00000000,"stop":0.00000000,"id":170625473,"initialAmount":0.50000000}]}}
    [scan_my_open_orders] 1 LTC/NIS open orders
    [scan_my_open_orders] Scanning 1 open LTC/NIS orders
    ***** [fetch_open_orders] request = {'pair': 'EtcNis'}
    
    Request: GET https://bit2c.co.il/Order/MyOrders {'Content-Type': 'application/x-www-form-urlencoded', 'key': 'REMOVED', 'sign': '+3uU29m/kwvB//++35kf9ESQ==', 'User-Agent': 'python-requests/2.22.0', 'Accept-Encoding': 'gzip, deflate'} nonce=1560112153&pair=EtcNis
    Response: GET https://bit2c.co.il/Order/MyOrders 200 {'Date': 'Sun, 09 Jun 2019 20:29:13 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '167', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=; expires=Mon, 08-Jun-20 20:29:13 GMT; path=/; domain=.bit2c.co.il; HttpOnly; Secure, .ASPXAUTH=05897B89974510E8DFF04D1C8419AE0893BDE7B7286E06976469A150DA403AA53C9C11B424935D8691A6F0102104E06F8758CCF2A2B0647FE971943019E9920C9C10A5BB0514C7A396EF3CA71D9909020E278C116C62FCBFCFC89CBC9B46F15EDEFCADD356C23A66FB29E0D825000A4A2E7CA37A6C445976FCAC71FB10DA6229AF4EA415DA5BB031741B9DD8A7B0FC09; path=/; HttpOnly', 'Cache-Control': 'private, s-maxage=0', 'Content-Encoding': 'gzip', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4e45db3f2dc897de-FRA'} {"EtcNis":{"ask":[{"created":1559672146,"type":1,"order_type":0,"amount":2.00000000,"price":36.63000000,"stop":0.00000000,"id":170950795,"initialAmount":2.00000000}],"bid":[{"created":1559672044,"type":0,"order_type":0,"amount":2.00000000,"price":25.78000000,"stop":0.00000000,"id":170950165,"initialAmount":2.00000000}]}}
    [scan_my_open_orders] 2 ETC/NIS open orders
    [scan_my_open_orders] Scanning 2 open ETC/NIS orders
    ***** [fetch_open_orders] request = {'pair': 'BtgNis'}
    
    Request: GET https://bit2c.co.il/Order/MyOrders {'Content-Type': 'application/x-www-form-urlencoded', 'key': 'REMOVED', 'sign': '/YQc9Hnoctqf4/==', 'User-Agent': 'python-requests/2.22.0', 'Accept-Encoding': 'gzip, deflate'} nonce=1560112156&pair=BtgNis
    Response: GET https://bit2c.co.il/Order/MyOrders 200 {'Date': 'Sun, 09 Jun 2019 20:29:16 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '199', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=; expires=Mon, 08-Jun-20 20:29:16 GMT; path=/; domain=.bit2c.co.il; HttpOnly; Secure, .ASPXAUTH=87DE52490C43844ACDBDA0B3147A5D792999F278D534E0A03FB13D40882BC1ED88859D21908BABDDA5D46C77829DEBC90BD0D5E5E49CB52C2C1E8A5EF78C811B8AC12378B8C4F7CB0F4C065EC67D1CD2A6208D38530DDAF06A7D486E5520BA3FB334E55F27012E6F765E727E2FA1C3192584CD59AC63B08BDB3A5A965A79E19087A52C6C5287940F909F3BADF5B12614; path=/; HttpOnly', 'Cache-Control': 'private, s-maxage=0', 'Content-Encoding': 'gzip', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4e45db51fd8a97de-FRA'} {"BtgNis":{"ask":[{"created":1559638089,"type":1,"order_type":0,"amount":0.50000000,"price":99.93000000,"stop":0.00000000,"id":170735518,"initialAmount":0.50000000}],"bid":[{"created":1559623700,"type":0,"order_type":0,"amount":1.00000000,"price":60.00000000,"stop":0.00000000,"id":170625722,"initialAmount":1.00000000},{"created":1559824962,"type":0,"order_type":0,"amount":1.00000000,"price":82.58000000,"stop":0.00000000,"id":171948983,"initialAmount":1.00000000}]}}
    [scan_my_open_orders] 3 BTG/NIS open orders
    [scan_my_open_orders] Scanning 3 open BTG/NIS orders
    ***** [fetch_open_orders] request = {'pair': 'BchSvNis'}
    
    Request: GET https://bit2c.co.il/Order/MyOrders {'Content-Type': 'application/x-www-form-urlencoded', 'key': 'REMOVED', 'sign': '+kmu+lLQ/=', 'User-Agent': 'python-requests/2.22.0', 'Accept-Encoding': 'gzip, deflate'} nonce=1560112159&pair=BchSvNis
    Response: GET https://bit2c.co.il/Order/MyOrders 200 {'Date': 'Sun, 09 Jun 2019 20:29:19 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '235', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=; expires=Mon, 08-Jun-20 20:29:19 GMT; path=/; domain=.bit2c.co.il; HttpOnly; Secure, .ASPXAUTH=00AE2B0C2C39B2C2EE7FF408BCD7CB93AD4D3EA05364C420F76859EA84FC36288EE7C08BB0B2FD68168AA5495164A6B43234E666F1700B828060C6EB2A977DB846FD6E4D1E5844429471670AD2E2C127CC0AB1BB9050D41B0550D2D441BF9F4F3E5A8CD0C958BD65598270B906ABBDAAEE68A3847848DA3407373C1B2FE8D69BF1DAD4E478D9EE14DBCDD7AD34476F28; path=/; HttpOnly', 'Cache-Control': 'private, s-maxage=0', 'Content-Encoding': 'gzip', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4e45db64bf6d97de-FRA'} {"BchsvNis":{"ask":[{"created":1559929677,"type":1,"order_type":0,"amount":0.05000000,"price":711.81000000,"stop":0.00000000,"id":172557907,"initialAmount":0.05000000}],"bid":[{"created":1559190994,"type":0,"order_type":0,"amount":1.00000000,"price":400.00000000,"stop":0.00000000,"id":168006613,"initialAmount":1.00000000},{"created":1559638133,"type":0,"order_type":0,"amount":0.50000000,"price":424.21000000,"stop":0.00000000,"id":170735836,"initialAmount":0.50000000},{"created":1559671968,"type":0,"order_type":0,"amount":0.10000000,"price":520.16000000,"stop":0.00000000,"id":170949739,"initialAmount":0.10000000}]}}
    [scan_my_open_orders] 0 BSV/NIS open orders
    [scan_my_open_orders] Scanning 0 open BSV/NIS orders
    ***** [fetch_open_orders] request = {'pair': 'GrinNis'}
    
    Request: GET https://bit2c.co.il/Order/MyOrders {'Content-Type': 'application/x-www-form-urlencoded', 'key': 'REMOVED', 'sign': 'uBUMVcwZ/+==', 'User-Agent': 'python-requests/2.22.0', 'Accept-Encoding': 'gzip, deflate'} nonce=1560112162&pair=GrinNis
    Response: GET https://bit2c.co.il/Order/MyOrders 200 {'Date': 'Sun, 09 Jun 2019 20:29:22 GMT', 'Content-Type': 'application/json; charset=utf-8', 'Content-Length': '340', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=; expires=Mon, 08-Jun-20 20:29:22 GMT; path=/; domain=.bit2c.co.il; HttpOnly; Secure, .ASPXAUTH=4DA652593ED571B5EB5C154C526BC045A8DC391AB383DC08978F87CC6105BD873306725081C8DEF63030DBC3CBA97C54E1F0129556FEAE956424260403CE48984A3430F4DEBF592531AB5D2FE8A1773112B82578048A8042DEA9165A8B4FA6B55DD5E9275D1EB4A57A35EA1F5B14C6B9B13ACEBA0CC26182F2DC1E0395294B65C6CBFC41C678DBEDDA86B1D85F068837; path=/; HttpOnly', 'Cache-Control': 'private, s-maxage=0', 'Content-Encoding': 'gzip', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4e45db7788c397de-FRA'} {"GrinNis":{"ask":[{"created":1559929767,"type":1,"order_type":0,"amount":1.00000000,"price":39.60000000,"stop":0.00000000,"id":172558253,"initialAmount":1.00000000},{"created":1559881705,"type":1,"order_type":0,"amount":1.00000000,"price":44.55000000,"stop":0.00000000,"id":172289299,"initialAmount":1.00000000},{"created":1559825179,"type":1,"order_type":0,"amount":1.00000000,"price":49.01000000,"stop":0.00000000,"id":171950366,"initialAmount":1.00000000},{"created":1559754314,"type":1,"order_type":0,"amount":2.00000000,"price":49.50000000,"stop":0.00000000,"id":171494912,"initialAmount":2.00000000},{"created":1556093360,"type":1,"order_type":0,"amount":13.17763154,"price":100.00000000,"stop":0.00000000,"id":150270323,"initialAmount":13.17763154}],"bid":[{"created":1556866354,"type":0,"order_type":0,"amount":32.83833333,"price":6.00000000,"stop":0.00000000,"id":155615840,"initialAmount":32.83833333},{"created":1558810794,"type":0,"order_type":0,"amount":5.00000000,"price":15.00000000,"stop":0.00000000,"id":166444069,"initialAmount":5.00000000},{"created":1559191033,"type":0,"order_type":0,"amount":3.06361482,"price":21.10000000,"stop":0.00000000,"id":168006798,"initialAmount":5.00000000},{"created":1559929750,"type":0,"order_type":0,"amount":1.00000000,"price":22.22000000,"stop":0.00000000,"id":172558170,"initialAmount":1.00000000}]}}
    [scan_my_open_orders] 9 GRIN/NIS open orders
    [scan_my_open_orders] Scanning 9 open GRIN/NIS orders

**Comment by kroitor** [comment]
Ok, from your verbose info the cause of the bug is immediately obvious:

- They had `BchabcNis`, we had `BchAbcNis` market id
- They had `BchsvNis`, we had `BchsvNis` market id

I've fixed the market ids for `bit2c` on the CCXT side, and if you update to 1.18.663 in 10 minutes – this bug should be resolves from there. Let us know if that doesn't help.

**Comment by asafravid** [comment]
Ok, waiting the 10 minutes and retesting

**Comment by asafravid** [comment]
Confirmed fix in `ccxt-1.18.663`
Thanks!

**Comment by kroitor** [comment]
@asafravid thx for the feedback!
